src/main/java/uk/gov/hmcts/cmc/claimstore/services/SealedClaimDocumentService.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.documents.CitizenSealedClaimPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.LegalSealedClaimPdfService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+@Service
+public class SealedClaimDocumentService {
+    private final ClaimService claimService;
+    private final LegalSealedClaimPdfService legalSealedClaimPdfService;
+    private final CitizenSealedClaimPdfService citizenSealedClaimPdfService;
+
+    @Autowired
+    public SealedClaimDocumentService(
+        final ClaimService claimService,
+        final LegalSealedClaimPdfService legalSealedClaimPdfService,
+        final CitizenSealedClaimPdfService citizenSealedClaimPdfService
+    ) {
+        this.claimService = claimService;
+        this.legalSealedClaimPdfService = legalSealedClaimPdfService;
+        this.citizenSealedClaimPdfService = citizenSealedClaimPdfService;
+    }
+
+    public byte[] generateLegalSealedClaim(final String claimExternalId, final String authorisation) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "authorisation". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1172)